yarn.lock

```diff
@@ -1146,11 +1146,6 @@
     mini-css-extract-plugin "0.4.3"
```

Comment:
Is it necessary to update the yarn.lock file?

Reply:
Don't need that. Just a result of me running `yarn install` with the `yarn 1.22`.